docs/csharp/programming-guide/interfaces/index.md

```diff
@@ -21,7 +21,7 @@ The name of the struct must be a valid C# [identifier name](../inside-a-program/
 
 Any class or struct that implements the <xref:System.IEquatable%601> interface must contain a definition for an <xref:System.IEquatable%601.Equals%2A> method that matches the signature that the interface specifies. As a result, you can count on a class that implements `IEquatable<T>` to contain an `Equals` method with which an instance of the class can determine whether it's equal to another instance of the same class.  
   
-The definition of `IEquatable<T>` doesn’t provide an implementation for `Equals`. The interface defines only the signature. In that way, an interface in C# is similar to an abstract class in which all the methods are abstract. However, a class or struct can implement multiple interfaces, but a class can inherit only a single class, abstract or not.
+The definition of `IEquatable<T>` doesn’t provide an implementation for `Equals`. The interface defines only the signature. In that way, an interface in C# is similar to such an abstract class in which all the methods are abstract. However, a class or struct can implement multiple interfaces, but a class can inherit only a single class, abstract or not.
```

Comment:
What you wrote here is fine. However, I'm suggesting a change because of updates we need to make in the C# 8.0 timeframe when default interface members are added to the language:

```suggestion
The definition of `IEquatable<T>` doesn’t provide an implementation for `Equals`. A class or struct can implement multiple interfaces, but a class can inherit only a single class.
```